lib/validation.js

```diff
@@ -61,8 +74,19 @@ function validate (handle, params, body, query) {
   return true
 }
 
-function serialize (handle, data) {
-  return handle[outputSchema](data)
+function serializeAccordingToResponseSchema (responseSchema, data, statusCode) {
+  if (responseSchema[statusCode]) {
+    return responseSchema[statusCode](data)
+  }
+  var fallbackStatusCode = (statusCode + '')[0] + 'xx'
+  if (responseSchema[fallbackStatusCode]) {
+    return responseSchema[fallbackStatusCode](data)
+  }
+  return responseSchema.default(data)
```

Comment:
Probably store `fast-safe-strinfgify` inside `default` is not needed, we can just call `fastSafeStringify` directly here. @mcollina thoughts?